packages/common/openapi.yml

```diff
@@ -0,0 +1,132 @@
+openapi: 3.0.3
+info:
+  title: Volunteer Victoria - Community
+  version: "1.0"
+paths:
+  /opportunity:
+    get:
+      responses:
+        '200':
+          description: Summary of opportunities
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/OpportunitySummary"
+                  
+    post:
+      requestBody:
+        description: Create a new opportunity
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/Opportunity"
+      responses:
+        "200":
+          description: ok
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Opportunity"
+        
+  /opportunity/{opportunityId}:
+    get:
+      parameters:
+      - name: opportunityId
+        in: path
+        required: true
+        schema:
+          type: string
+      responses:
+        '200':
+          description: Opportunity
+          content:
+            application/json:
+              schema: 
+                $ref: "#/components/schemas/Opportunity"
+
+    put:
+      parameters:
+      - name: opportunityId
+        in: path
+        required: true
+        schema:
+          type: string 
+      requestBody:
+        description: Update an existing opportunity
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/Opportunity"
+      responses:
+        '200':
+          description: Opportunity
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Opportunity"
+
+  /user:
+    get:
+      responses:
+        '200':
+          description: User
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/User"
+  
+components:
+  schemas:
+    OpportunitySummary:
+      type: object
+      properties:
+        opportunityId:
+          type: string
+        title:
+          type: string
+        contactName:
+          type: string
+        requiredPeopleCount:
+            type: number
+        startTime:
+          type: number
+          example: 1666817899
```

Comment:
Can we talk with ISO 8601 date strings?

Reason - bakes the TZ right in for easier handling.

Reply:
As discussed in separate chat, we are going to go with epoch time for this project.